django_x509/base/models.py

```diff
@@ -197,37 +198,37 @@ def save(self, *args, **kwargs):
 
     @cached_property
     def x509(self):
-        """
+        '''
```

Comment:
was it you who changed these docstrings or did you use black?

Avoid changing these lines please.

Reply:
I was experimenting with a flake8 extension before we started with moving to black, but I think at this point, I can reverse it and perhaps if black wants it we can do it in a separate pull request! :smile:

Reply:
I will get back to it asap! :smile: